packages/strapi-plugin-i18n/admin/src/containers/SettingsPage/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
This component should be very similar to the `webhooks` one. Can we create one in order to simplify the codebase?